pkg/injection/namespace/mapper/mapper_test.go

```diff
@@ -211,4 +211,22 @@ func TestUpdateNamespace(t *testing.T) {
 		require.True(t, updated)
 		assert.Empty(t, namespace.Labels)
 	})
+	t.Run("Do not inject into pods when adding namespace to ignored namespaces", func(t *testing.T) {
```

Comment:
nitpick:
```suggestion
	t.Run("Remove injection label from ignored-namespaces if present from previous setup", func(t *testing.T) {
```

Reply:
fixed in 7ddca951561e58b96cc169fbc6bcda71eb7857d0